### DISCUSSION TITLE
`oidc-client` package take over

### DISCUSSION THREAD

**Discussion by aheimlich** [panva/openid-client#417] [Q&A] [answered]
Hi,

According to https://github.com/IdentityModel/oidc-client-js/issues/1391#issuecomment-853274692, you expressed interested in taking over maintenance of the [`oidc-client` package](https://www.npmjs.com/package/oidc-client). Did anything ever come of this?

**Comment by panva** [comment, marked as answer]
I have since retracted my expressed interest.
